Metric_practice.py

```diff
@@ -1,50 +1,56 @@
 import numpy as np
 
 # mock feature matrix
-feature = np.array([  [4, 2, 6],
-                      [4, 9, 6],
-                      [2, 5, 0],
-                      [2, 0, 9],
-                      [5, 3, 0]])
+feature = np.array([[4, 2, 6],
+                    [4, 9, 6],
+                    [2, 5, 0],
+                    [2, 0, 9],
+                    [5, 3, 0]])
 
 
 def euclidean_distance(x, y):
-    """
-    Computes the euclidean distance between two vectors
-    ----------
+    """The distance between two vectors
+    
+    Computs the eucladian distance between vectors x and y
+    
     Parameters
     ----------
-    x : first vector
-    y : second vector
-    -------
+    x : array_like
+        the first feature vector
+    y : array_like
+        the second feature vector
+   
     Returns
     -------
-    distance : euclidan distance between x and y
+    distance : ndarray
+        the euclidean distance between the two vectors
     """
-    intermdiate = 0
-    for i in range(0, len(x)):
-        intermdiate += ((x[i] - y[i])**2)
-    distance = np.sqrt(intermdiate)
+    distance = np.sqrt(sum(((x - y)) ** 2))
     return distance
 
 
-def pairwaise_distance(matrix):
-    """
-    Computes the pairwise distance between each vectore
-    in a given matrix
-    ----------
+def pairwise_distance(Z):
+    """Pairwise distance between vectors
+
+    Computes the pairwise distance between each vector
+    in a given array
+    
     Parameters
     ----------
-    matrix : given matrix
-    -------
+    Z : ndarray
```

Comment:
We usually don't have a capitalized letter as an argument name. Low case is a good practice.